openmetadata-service/src/main/java/org/openmetadata/service/search/SearchRepository.java

```diff
@@ -298,6 +298,31 @@ public void createEntity(EntityInterface entity) {
     }
   }
 
+  public void createEntities(List<EntityInterface> entities) {
```

Comment:
Lets tag this method as @Transaction

Reply:
IMO , where we are are inserting into DB and calling this method should be wrapped in a method and tag with @transaction that way if ES fails it can rollback the Database

Reply:
Ignore above, looks like its handled